### ISSUE TITLE
[0.9.2.4] - TargetInvocationException - On wealth value

### ISSUE THREAD

**Issue by fillion07** [StrangeLoopGames/EcoIssues#20475]
Exception I get after force finish election on an executive order:
![image](https://user-images.githubusercontent.com/7052045/107527570-ecf1ee80-6b86-11eb-8c7a-717403e57cf4.png)

[Logs.zip](https://github.com/StrangeLoopGames/EcoIssues/files/5959084/Logs.zip)
[Dump.zip](https://github.com/StrangeLoopGames/EcoIssues/files/5959086/Dump.zip)

```
TargetInvocationException

[09:50:37.911] [447] [Error] [Eco] System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at Eco.Gameplay.Systems.Chat.ChatCommandService.<>c__DisplayClass12_0.<InvokeCommand>b__1()
   at Eco.Core.Utils.Async.TaskUtils.<>c__DisplayClass2_0.<RunWithExceptionLog>b__0()
System.NullReferenceException: Object reference not set to an instance of an object.
   at System.Object.GetType()
   at Eco.Gameplay.Civics.GameValues.GameValueContext`1.Value(IContextObject context)
   at Eco.Gameplay.Civics.GameValues.Values.Wealth.Value(IContextObject action)
   at Eco.Gameplay.Civics.MoneyTransfer_LegalAction.GetAndCheckValues(IContextObject context)
   at Eco.Gameplay.Civics.TransferToAccount_LegalAction.GetAndCheckValues(IContextObject context)
   at Eco.Gameplay.Civics.MoneyTransfer_LegalAction.Do(LocString description, IContextObject action, Law law)
   at Eco.Gameplay.Civics.MoneyTransfer_LegalAction.Eco.Gameplay.Civics.Laws.ExecutiveActions.IExecutiveAction.PerformExecutiveAction(User user, IContextObject context)
   at Eco.Gameplay.Civics.Laws.ExecutiveActions.CivicAction_PerformExecutiveAction.PerformBase(User executingUser)
   at Eco.Gameplay.Civics.Laws.ExecutiveActions.CivicAction_PerformExecutiveAction.Perform(User executingUser, Election election)
   at Eco.Gameplay.Civics.Elections.Election.ApplyIfFinished(Boolean forceEnd)
   at Eco.Gameplay.Civics.CivicsCommands.FinishElection(User user, Election election, Boolean addVote)
   ```

**Comment by TwixterZ** [comment]
Thank you for reporting this @fillion07 can you share the law you have created that is causing this exception for us to re-create?

**Comment by fillion07** [comment]
![image](https://user-images.githubusercontent.com/7052045/110596597-7b896980-814d-11eb-9f84-ff12b6a72214.png)
It was something like this law in the Executive Office in Actions to run once. For some reason I wasn't able to reproduce the error message. But the action wasn't executed when I force finished the election for that law.

My goal was to transfer the entire wealth of a player 1 (in store credit of player 2) to player 2.

**Comment by IcedForge** [comment]
There were a lot of fixes for transactions in the recent 9.3 update, are you still having this issue after updating?

**Comment by TwixterZ** [comment]
Re-test performed for this issue and it is working as of 9.3.5 Release 231. 

Closing this issue
